WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -47,8 +47,11 @@ public class KeyManager
 		public string FilePath { get; private set; }
 		private object ToFileLock { get; }
 
+		[JsonProperty(Order = 5)]
+		public bool? PasswordVerified { get; private set; }
+
 		[JsonConstructor]
-		public KeyManager(BitcoinEncryptedSecretNoEC encryptedSecret, byte[] chainCode, ExtPubKey extPubKey, string filePath = null)
+		public KeyManager(BitcoinEncryptedSecretNoEC encryptedSecret, byte[] chainCode, ExtPubKey extPubKey, bool? passwordVerified, string filePath = null)
```

Comment:
Shouldn't it cause issues? Should we default this to something?

Reply:
I mean at deserialization of old wallet files for example.

Reply:
PasswordVerified will be deserialized to null. And password Test triggered on false:
https://github.com/zkSNACKs/WalletWasabi/pull/1317/files#diff-0e18af0672330f941fb821c9749f5763R256